group-generators/generators/example-dynamic-graphql/index.ts

```diff
@@ -0,0 +1,116 @@
+import { gql } from "graphql-request";
+import {
+  DynamicGraphQLProvider,
+  DynamicGraphQLType,
+} from "@group-generators/helpers/data-providers/dynamic-graphql";
+import { Tags, ValueType, GroupWithData, AccountSource } from "topics/group";
+import {
+  GenerationContext,
+  GenerationFrequency,
+  GroupGenerator,
+} from "topics/group-generator";
+
+const generator: GroupGenerator = {
+  generationFrequency: GenerationFrequency.Once,
+  generate: async (context: GenerationContext): Promise<GroupWithData[]> => {
+    const dynamicGraphQLProvider = new DynamicGraphQLProvider();
+    const ENDPOINT = "https://hub.snapshot.org/graphql";
+    const JMESPATH = `votes[].voter`;
+    const GRAPHQL_QUERY: string = gql`
```

Comment:
For the query, normally, if the group generator comes from the Factory, the query will have this format:
"query VotesQuery { votes ( first: 1000 skip: 0 where: { proposal: \"0x123\"} orderBy:..."
And it works so it's good 👌

But there is a problem that does not depend on you, indeed the Factory forbids the use of double quotes "" on string types (UX reasons), and because we need them in the query it cause an issue.

But I think a JSON type would do the job, I'll try it